tests/chassis_modules_test.py

```diff
@@ -202,6 +259,41 @@ def test_config_startup_module(self):
         result_out = " ".join((result_lines[header_lines]).split())
         assert result_out.strip('\n') == show_linecard0_startup_output.strip('\n')
 
+    def test_config_startup_module_fabric(self):
```

Comment:
I would suggest to add test cases for the following scenarios as well.
1. FC is up and doing startup
2. FC is down and doing startup
3. FC not present.